### ISSUE TITLE
Request: show Result when approval is completed

### ISSUE THREAD

**Issue by StefanS365** [aboodh95/ApprovalStatusPCF#2]
Request for change:
Please show the value of the Result field in the control when the approval is completed.
![image](https://user-images.githubusercontent.com/33868400/83044579-a84c1480-a044-11ea-9a5a-6e78a64ffbf3.png)

**Comment by aboodh95** [comment]
Hi @StefanS365 ,

Once I build the control, I had this idea, but I have some cases where you might have more than one approval, or you have more than one flow triggered (2 steps approval), so what I say that for sure you are having some fields in your entity you are updating after the approval changed, anyhow I will work on it

**Comment by StefanS365** [comment]
Agree, but you could show the Status Reason / Result of the latest / most recent Approval that is related to the record via the item link.
